contrib/pyln-spec/bolt7/pyln/spec/bolt7/text.py

```diff
@@ -286,7 +286,6 @@
          onion service addresses; Encodes:
          `[32:32_byte_ed25519_pubkey] || [2:checksum] || [1:version]`, where
          `checksum = sha3(".onion checksum" | pubkey || version)[:2]`.
-   * `5`: DNS hostname; data = `[byte:len][len*byte:hostname][u16:port]` (length up to 258)
 
```

Comment:
Use a separate comment for this section, i.e:

```
/* BOLT-bolt7-hostnames #7: 
 *     * `5`: DNS hostname; data = `[byte:len][len*byte:hostname][u16:port]` (length up to 258)
 */
```

Then you can check just those quotes with:
```
make check-source-bolt BOLTVERSION=bolt7-hostnames
```

(Or use a real git-id instead of a branch name, if you prefer!)